Library/Homebrew/test/string_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "extend/string"
+
+describe String do
+  describe "#undent" do
+    it "removes leading whitespace, taking the first line as reference" do
+      expect(
+        <<-EOS.undent
```

Comment:
Homebrew convention seems to be to line it up with the indentation of its contents, e.g. https://github.com/Homebrew/homebrew-core/blob/fa4f85a5712dabdcc89522a8fd93dcb3797d1e57/Formula/awscli.rb